Earthfile

```diff
@@ -194,7 +194,7 @@ rust:
   RUN apk add --no-cache rust
 
   COPY ./src/leibniz.rs ./
-  RUN --no-cache export RUST_BACKTRACE=1; rustc -C debuginfo=0 -C opt-level=3 leibniz.rs
+  RUN --no-cache export rustc -C debuginfo=0 -C opt-level=3 -C target-cpu=native leibniz.rs
```

Comment:
This is likely to have the most significant impact. Otherwise, the machine that LLVM will optimize for will not be the current machine, as done for C and C++, but will be a potentially much older one, limited to using only a ~20 year old instruction set.

Reply:
build-std, lto, panic=abort etc also makes some minor differences (ref #24). Couldn't get rustc to do build-std. And anyway, maybe it's not fair.